homeassistant/components/ecobee/repairs.py

```diff
@@ -0,0 +1,38 @@
+"""Repairs support for Ecobee."""
+
+from __future__ import annotations
+
+from homeassistant.components.repairs import RepairsFlow
+from homeassistant.components.repairs.issue_handler import ConfirmRepairFlow
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers import issue_registry as ir
+
+from .const import DOMAIN
+
+
+@callback
+def migrate_notify_issue(hass: HomeAssistant, entry: ConfigEntry) -> None:
+    """Ensure an issue is registered."""
+    ir.async_create_issue(
+        hass,
+        DOMAIN,
+        "migrate_notify",
+        breaks_in_ha_version="2024.11.0",
+        issue_domain=Platform.NOTIFY.value,
```

Comment:
I think we can import the DOMAIN from homeassistant.components.notify instead, looks cleaner